doc-content/enterprise-only/release-notes/rn-kogito-productize-features-ref.adoc

```diff
@@ -0,0 +1,17 @@
+[id='rn-kogito-productize-features-ref']
+= Kogito productized features in {PRODUCT}
+
+{PRODUCT} {ENTERPRISE_VERSION} provides the following Kogito based applications that you can use to design Business Process Model and Notation (BPMN) process models and Decision Model and Notation (DMN) decision models using graphical modelers:
```

Comment:
We need a way to express that the VS Code tool is not only for Kogito, but also for regular PAM/DM projects.

Those editors were developed under Kogito initiativbe, but are useful for PAM and Kogito.

Reply:
Hey @porcelli, Apart from **Kogito productized features** section I have added a separate **3.3. Support for VSCode extension** section in the release notes and all the details are included in that section.

Reply:
I understand, but by a user that reads the above sentence would be confuse on what exactly VS Code extensions are expected to support. I'd suggest some better clarification.

Reply:
@porcelli, Apart from Kogito productized features section, I have added a separate **3.3. Support for VSCode extension section** in the same release notes and all the details are included in that section.